src/reporters/build_kite.ts

```diff
@@ -0,0 +1,183 @@
+    journey: Journey,
+    { error, start, end, status }: JourneyEndResult
+  ) {
+    const { failed, succeeded, skipped } = this.metrics;
+    const total = failed + succeeded + skipped;
+    if (total === 0 && error) {
+      this.write(renderError(error));
+    }
+    const message = `${symbols[status]} Took  (${renderDuration(
+      end - start
+    )} seconds)`;
+    this.write(message);
+
+    const FAILED_STEPS_PATH = join(CACHE_PATH, 'failed_steps');
+
+    await fs.promises.mkdir(FAILED_STEPS_PATH, { recursive: true });
+
+    await gatherScreenshots(
+      join(CACHE_PATH, 'screenshots'),
+      async screenshot => {
+        const { data, step } = screenshot;
+
+        if (status === 'failed') {
+          await (async () => {
+            await fs.promises.writeFile(
+              join(FAILED_STEPS_PATH, `${step.name}.jpg`),
+              data,
+              {
+                encoding: 'base64',
+              }
+            );
+          })();
+        }
+      }
+    );
+  }
+
+  onEnd() {
+    const failedJourneys = Array.from(this.journeys.entries()).filter(
+      ([, steps]) => steps.some(step => step.status === 'failed')
+    );
+
+    if (failedJourneys.length > 0) {
+      failedJourneys.forEach(([journeyName, steps]) => {
+        const name = red(`Journey: ${journeyName} 🥵`);
+        this.write(`\n+++ ${name}`);
+        steps.forEach(stepResult => {
+          const { status, end, start, error, name: stepName } = stepResult;
+          const message = `${
+            symbols[status]
+          }  Step: '${stepName}' ${status} (${renderDuration(
+            (end - start) * 1000
+          )} ms)`;
+          this.write(indent(message));
+          if (error) {
+            this.write(renderError(error));
+          }
+        });
+      });
+    }
+
+    const successfulJourneys = Array.from(this.journeys.entries()).filter(
+      ([, steps]) => steps.every(step => step.status === 'succeeded')
+    );
+
+    const dirPath = this.recordVideo?.dir;
+    if (dirPath) {
+      successfulJourneys.forEach(([journeyName]) => {
+        const filePath = dirPath + journeyName + '.webm';
+        try {
+          fs.unlinkSync(filePath);
+        } catch (e) {
+          // eslint-disable-next-line no-console
+          console.log('Failed to delete video file for path ' + filePath);
+        }
+      });
+    }
+
+    const { failed, succeeded, skipped } = this.metrics;
+    const total = failed + succeeded + skipped;
+
+    let message = '\n';
+    if (total === 0) {
+      message = 'No tests found!';
+      message += ` (${renderDuration(now())} ms) \n`;
+      this.write(message);
+      return;
+    }
+
+    message += succeeded > 0 ? green(` ${succeeded} passed`) : '';
+    message += failed > 0 ? red(` ${failed} failed`) : '';
+    message += skipped > 0 ? cyan(` ${skipped} skipped`) : '';
+    message += ` (${renderDuration(now() / 1000)} seconds) \n`;
+    this.write(message);
+  }
+
+  write(message: any) {
```

Comment:
Not sure why you are overriding this, we should let the base reporter handle this.